src/heliumdb/module.cpp

```diff
@@ -371,15 +368,15 @@ heliumdb_ass_sub (heliumdbPy* self, PyObject* k, PyObject* v)
         PyErr_SetString (HeliumDbException, err);
         return -1;
     }
-
     return 0;
 }
 
 
+static char   buffer[700000];
```

Comment:
Hi, why did you make the buffer static out here I don't see why there was a memory leak with using the stack buffer?

Reply:
This particular change was made for performance enhancing reasons.